client/app/components/PermissionsEditorDialog/index.jsx

```diff
@@ -170,8 +170,10 @@ function PermissionsEditorDialog({ dialog, author, context, aclUrl }) {
                 ) : (
                   <Tooltip title="Remove user permissions">
                     <i
-                      className="fa fa-remove clickable"
+                      role="button"
+                      tabIndex={-1}
```

Comment:
Why is this `tabIndex` -1? Should it be `0`?

Reply:
Will track it to the other PR, the proper way would be to wrap this in a styleless button

Reply:
If we're giving it a role of `button` and a click handler, we should at least give it `tabIndex={0}` so a user can get to it.